style.css

```diff
@@ -359,3 +359,96 @@ button:active {
 .drop-down:active {
   background-color: #7a869a;
 }
+
```

Comment:
Your form elements seem to be causing a lot of extra horizontal whitespace in your design. Consider modifying the top of your CSS file to include the following code in order to eliminate the whitespace and make your design match the figma design

```
* {
  box-sizing: border-box;
}

body {
  margin: 0;
  padding: 0;
  font-family: "Poppins", sans-serif;
}

```

Your design:
<img width="237" alt="image" src="https://github.com/Kwamay/Porfolio_Setup_Project/assets/37116322/fe6fdb26-d6d2-4c21-802e-a05c40bd6203">

Figma design:
<img width="213" alt="image" src="https://github.com/Kwamay/Porfolio_Setup_Project/assets/37116322/81553028-922e-4f12-8d3e-0cd04aa70e61">